cmake/projects/OpenBLAS/hunter.cmake

```diff
@@ -32,15 +44,20 @@ hunter_add_version(
 )
 
 hunter_configuration_types(OpenBLAS CONFIGURATION_TYPES Release)
-if (MSVC)
+if(MSVC)
   hunter_pick_scheme(DEFAULT url_sha1_cmake)
-else()
+elseif(HUNTER_OpenBLAS_VERSION VERSION_LESS 0.3.0)
   hunter_pick_scheme(DEFAULT OpenBLAS)
+else()
+  hunter_cmake_args(
+    OpenBLAS
+    CMAKE_ARGS
+    NOFORTRAN=1
+  )
+  hunter_pick_scheme(DEFAULT url_sha1_cmake)
 endif()
 hunter_cacheable(OpenBLAS)
 hunter_download(
     PACKAGE_NAME OpenBLAS
     PACKAGE_INTERNAL_DEPS_ID "2"
-    PACKAGE_UNRELOCATABLE_TEXT_FILES
```

Comment:
This change will break old versions like `0.2.20-p0`.

As far as I can see since version `0.3.0-p2` we will not need `OpenBLAS` custom build scheme, right? In this case I'm okay with removing `0.2.*` versions since changing custom build scheme to CMake based build scheme is a super cool event :) We will not need `PACKAGE_INTERNAL_DEPS_ID` and `PACKAGE_UNRELOCATABLE_TEXT_FILES` in this case too.

Reply:
> Or is there a way to make the package_unrelocatable_text_files depend on some variable?

```cmake
if(HUNTER_OpenBLAS_VERSION VERSION_LESS 0.3.0)
  set(
      _openblas_unrelocatable_text_files
      PACKAGE_UNRELOCATABLE_TEXT_FILES
      "lib/cmake/openblas/OpenBLASConfig.cmake
  )
else()
  set(_openblas_unrelocatable_text_files "")
endif()

hunter_download(
    PACKAGE_NAME OpenBLAS
    PACKAGE_INTERNAL_DEPS_ID "2"
    ${_openblas_unrelocatable_text_files}
)
```

Reply:
OK, I've done that. Also made MSVC consistent (it didn't work previously in 0.2.20 anyway).